copyright

```diff
@@ -1411,6 +1417,8 @@ Files:
  images/outfit/double?plasma?core*
  images/outfit/dual?sunbeam?turret*
  images/outfit/finisher?maegrolain*
+ images/outfit/*grav-steering*
+ images/outfit/*grav-thruster*
```

Comment:
Are these outdated? This PR doesn't seem to add any such files.

Reply:
Yup, should be `efreti?` not `grav-`